ui/src/data-products/translations.fi.json

```diff
@@ -0,0 +1,30 @@
+{
+    "data-products": {
+        "element-list": {
+            "element-list-title": "Elementtiluettelo",
+            "continuous-geometry": "Jatkuva geometria",
+            "plan-geometry": "Suunnitelman geometria",
+            "geometry-elements": "Geometriaelementit",
+            "element-list-table": {
+                "track-number": "Ratanumero",
+                "alignment": "Raide",
+                "element-type": "Elementin tyyppi",
+                "track-address-start": "Rataosoite alku",
+                "track-address-end": "Rataosoite loppu",
```

Comment:
Näin tosiaan luki Figman speksissä, mutta omasta mielestäni näissäkin voisi silti lukea muiden columnien tapaan "... alussa" ja "... lopussa"

Reply:
Totta, hyvä idea pitää yhtenäisenä. Korjattu.